packages/client/public/root-cookie-accessor-template.html

```diff
@@ -56,9 +56,15 @@
               break
             } else {
               const hasAccess = await document.hasStorageAccess()
-              const storageAccessPermission = await navigator.permissions.query({
-                name: 'storage-access'
-              })
+              let storageAccessPermission
+              try {
+                await navigator.permissions.query({
```

Comment:
I think we need to attempt to set storageAccessPermission with the result of this, otherwise if it succeeds, storageAccessPermission will still be null and the if statement on line 68 will never get triggered properly.

Reply:
Apologies for missing that earlier. That was my intention from the start.